docs/pages/enroll-resources/machine-id/introduction.mdx

```diff
@@ -49,6 +49,27 @@ access to and what sort of proof (known as the **join method**) is needed to use
 this join token. This proof is typically an identity issued to the machine by
 the platform it runs on (e.g. AWS IAM).
 
+Multiple join tokens may be created for a single bot to allow joining with
+different join methods.
+
+### Bot Instances
```

Comment:
```suggestion
### Bot instances
```

Reply:
hmm I was going off of the the previous entries of 'bot instances' which were not capitalized:

```
% grep -inr "bot instance" --include="*.mdx" */
enroll-resources//machine-id/troubleshooting.mdx:46:this internal data is otherwise being shared between multiple bot instances.
enroll-resources//machine-id/troubleshooting.mdx:58:Otherwise, first ensure only one bot instance is using the internal data
/Users/mmcallister/Teleport/teleport/docs/pages [master]
```

but maybe they should be? I'll retract this suggestion.  We should edit the older entries there as well for consistency though.